app/lib/aether/aether.coffee

```diff
@@ -0,0 +1,244 @@
+self = window if window? and not self?
```

Comment:
This file is the main aether file that has been copy pasted in.

Reply:
It is hard to know which of the public interface is still being used. So currently there is probably lots of excess code.